plugins/modules/snapshot.py

```diff
@@ -125,19 +157,29 @@ def main():
     module = ForemanSnapshotModule(
         argument_spec=dict(
             state=dict(default='present', choices=['present', 'absent', 'reverted']),
+            new_entity=dict(required=False, type='bool', default=False)
```

Comment:
Would it be possible to use a new `state` choice like `new` or `created` instead of a new parameter?

Reply:
Technically yes, but I am not sure what the better UX is, need to think about that a bit (and have time for that 😞)

Reply:
how can we proceed with this?

Reply:
Added a new state called `new_state`.